fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalOlapScan.java

```diff
@@ -103,14 +105,7 @@ public boolean equals(Object o) {
         if (this == o) {
             return true;
         }
-        if (o == null || getClass() != o.getClass() || !super.equals(o)) {
-            return false;
-        }
-        PhysicalOlapScan that = (PhysicalOlapScan) o;
-        return selectedIndexId == that.selectedIndexId
-                && Objects.equals(selectedTabletIds, that.selectedTabletIds)
-                && Objects.equals(selectedPartitionIds, that.selectedPartitionIds)
-                && Objects.equals(olapTable, that.olapTable);
+        return o != null && getClass() == o.getClass() && super.equals(o);
```

Comment:
why change this, i think we decide equals and hashCode should contains all attributes